internal/provider/networks_wireless_ssids_resource.go

```diff
@@ -0,0 +1,1727 @@
+	MinBitrate                       jsontypes.Float64        `tfsdk:"min_bitrate"`
+	UseVlanTagging                   jsontypes.Bool           `tfsdk:"use_vlan_tagging"`
+	DisassociateClientsOnVpnFailover jsontypes.Bool           `tfsdk:"disassociate_clients_on_vpn_failover"`
+	RadiusOverride                   jsontypes.Bool           `tfsdk:"radius_override"`
+	RadiusGuestVlanEnabled           jsontypes.Bool           `tfsdk:"radius_guest_vlan_enabled"`
+	Enabled                          jsontypes.Bool           `tfsdk:"enabled"`
+	RadiusProxyEnabled               jsontypes.Bool           `tfsdk:"radius_proxy_enabled"`
+	RadiusTestingEnabled             jsontypes.Bool           `tfsdk:"radius_testing_enabled"`
+	RadiusFallbackEnabled            jsontypes.Bool           `tfsdk:"radius_fallback_enabled"`
+	RadiusCoaEnabled                 jsontypes.Bool           `tfsdk:"radius_coa_enabled"`
+	RadiusAccountingEnabled          jsontypes.Bool           `tfsdk:"radius_accounting_enabled"`
+	LanIsolationEnabled              jsontypes.Bool           `tfsdk:"lan_isolation_enabled"`
+	Visible                          jsontypes.Bool           `tfsdk:"visible"`
+	AvailableOnAllAps                jsontypes.Bool           `tfsdk:"available_on_all_aps"`
+	MandatoryDhcpEnabled             jsontypes.Bool           `tfsdk:"mandatory_dhcp_enabled"`
+	AdultContentFilteringEnabled     jsontypes.Bool           `tfsdk:"adult_content_filtering_enabled"`
+	WalledGardenEnabled              jsontypes.Bool           `tfsdk:"walled_garden_enabled"`
+	Dot11W                           types.Object             `tfsdk:"dot11w"`
+	Dot11R                           types.Object             `tfsdk:"dot11r"`
+	LocalRadius                      types.Object             `tfsdk:"local_radius"`
+	Ldap                             types.Object             `tfsdk:"ldap"`
+	ActiveDirectory                  types.Object             `tfsdk:"active_directory"`
+	DnsRewrite                       types.Object             `tfsdk:"dns_rewrite"`
+	SpeedBurst                       types.Object             `tfsdk:"speed_burst"`
+	Gre                              types.Object             `tfsdk:"gre"`
+	Oauth                            types.Object             `tfsdk:"oauth"`
+	SplashGuestSponsorDomains        types.List               `tfsdk:"splash_guest_sponsor_domains"`
+	WalledGardenRanges               types.List               `tfsdk:"walled_garden_ranges"`
+	AvailabilityTags                 types.List               `tfsdk:"availability_tags"`
+	RadiusServers                    []RadiusServer           `tfsdk:"radius_servers"`
+	RadiusAccountingServers          []RadiusAccountingServer `tfsdk:"radius_accounting_servers"`
+	ApTagsAndVlanIds                 []ApTagsAndVlanId        `tfsdk:"ap_tags_and_vlan_ids"`
+}
+
+// NetworksWirelessSsidsResourceModel describes the resource data model.
+type NetworksWirelessSsidsResourceModel struct {
```

Comment:
See above comment on nested structs for all of the structs above this resource data model ^